[2023-10-31 23:57:35,057: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-10-31 23:57:35,058: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-10-31 23:57:41,470: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:41] "GET / HTTP/1.1" 200 -]
[2023-10-31 23:57:42,960: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:42] "GET /static/js/scripts.js HTTP/1.1" 200 -]
[2023-10-31 23:57:42,976: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:42] "GET /static/css2/style.css HTTP/1.1" 200 -]
[2023-10-31 23:57:42,976: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:42] "GET /static/css/styles.css HTTP/1.1" 200 -]
[2023-10-31 23:57:42,982: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:42] "GET /static/css2/nunito-font.css HTTP/1.1" 200 -]
[2023-10-31 23:57:43,901: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:43] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-10-31 23:57:43,905: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:43] "GET /static/assets/img/form-v9.jpg HTTP/1.1" 200 -]
[2023-10-31 23:57:43,990: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:43] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-10-31 23:57:44,195: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:57:44] "GET /static/assets/favicon.ico HTTP/1.1" 200 -]
[2023-10-31 23:58:22,679: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:22] "POST /predict HTTP/1.1" 200 -]
[2023-10-31 23:58:22,774: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:22] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2023-10-31 23:58:30,041: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "GET / HTTP/1.1" 200 -]
[2023-10-31 23:58:30,081: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-10-31 23:58:30,087: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-10-31 23:58:30,095: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-10-31 23:58:30,098: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-10-31 23:58:30,121: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-10-31 23:58:30,141: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-10-31 23:58:30,162: INFO: _internal: 192.168.0.100 - - [31/Oct/2023 23:58:30] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:10:04,575: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:10:04] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:34:59,819: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.100:8080]
[2023-11-01 00:34:59,821: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-11-01 00:35:19,886: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:19] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:35:23,301: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:23] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:35:29,800: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:29] "GET / HTTP/1.1" 200 -]
[2023-11-01 00:35:31,055: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:31,056: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:31,057: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:31,059: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:31,086: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 00:35:31,098: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:31,123: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:35:31,169: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:31] "[36mGET /static/assets/favicon.ico HTTP/1.1[0m" 304 -]
[2023-11-01 00:35:47,084: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:35:47] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:38:02,728: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.100:8080]
[2023-11-01 00:38:02,728: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-11-01 00:38:11,447: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:11] "GET / HTTP/1.1" 200 -]
[2023-11-01 00:38:12,965: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:12] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:38:12,965: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:12] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:38:12,965: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:12] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:38:12,982: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:12] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:38:13,023: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:13] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 00:38:13,028: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:13] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 00:38:13,131: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:13] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:38:31,522: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:38:31] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:41:29,770: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 00:41:29,775: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 00:41:29,781: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 00:41:29,785: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 00:41:29,787: INFO: common: created directory at: artifacts]
[2023-11-01 00:41:29,788: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 00:41:30,804: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 8D72:2B12C3:8CC0A:DFF24:65415160
Accept-Ranges: bytes
Date: Tue, 31 Oct 2023 19:11:28 GMT
Via: 1.1 varnish
X-Served-By: cache-del21748-DEL
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1698779489.515009,VS0,VE282
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 7ad949db08e2d1b62c08a02a15f7781c05a50ee4
Expires: Tue, 31 Oct 2023 19:16:28 GMT
Source-Age: 0

]
[2023-11-01 00:41:30,830: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-01 00:41:30,832: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-01 00:41:30,839: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 00:41:30,842: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 00:41:30,849: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 00:41:30,850: INFO: common: created directory at: artifacts]
[2023-11-01 00:41:30,852: INFO: common: created directory at: artifacts/data_validation]
[2023-11-01 00:41:30,891: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-01 00:41:30,897: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-01 00:41:30,912: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 00:41:30,915: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 00:41:30,920: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 00:41:30,922: INFO: common: created directory at: artifacts]
[2023-11-01 00:41:30,924: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-01 00:41:30,973: INFO: data_transformation: Splited data into training and test sets]
[2023-11-01 00:41:30,974: INFO: data_transformation: (1199, 12)]
[2023-11-01 00:41:30,976: INFO: data_transformation: (400, 12)]
[2023-11-01 00:41:30,976: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-01 00:41:30,976: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-01 00:41:30,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 00:41:30,991: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 00:41:30,996: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 00:41:30,998: INFO: common: created directory at: artifacts]
[2023-11-01 00:41:30,999: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-01 00:41:31,062: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2023-11-01 00:41:31,064: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2023-11-01 00:41:31,064: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 00:41:31,070: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 00:41:31,075: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 00:41:31,075: INFO: common: created directory at: artifacts]
[2023-11-01 00:41:31,079: INFO: common: created directory at: artifacts/model_evaluation]
[2023-11-01 00:41:31,405: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2023-11-01 00:41:42,091: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2023-11-01 00:42:06,894: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.100:8080]
[2023-11-01 00:42:06,894: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-11-01 00:42:11,762: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:11] "GET / HTTP/1.1" 200 -]
[2023-11-01 00:42:13,052: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:13,052: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:13,063: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:13,068: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:13,088: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 00:42:13,109: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:13,129: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:42:13,161: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:13] "[36mGET /static/assets/favicon.ico HTTP/1.1[0m" 304 -]
[2023-11-01 00:42:31,705: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:42:31] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:43:22,361: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-01 00:43:22,362: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-01 00:43:30,470: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:43:30] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:43:30,621: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:43:30] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-01 00:43:31,224: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:43:31] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:43:31,224: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:43:31] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:27,546: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-01 00:44:27,546: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-01 00:44:42,700: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:42] "GET / HTTP/1.1" 200 -]
[2023-11-01 00:44:44,497: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:44,519: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:44,551: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:44,558: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:44,575: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 00:44:44,701: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 00:44:44,768: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:44:44] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:45:10,765: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:45:10] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:45:10,829: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:45:10] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:45:10,830: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:45:10] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:45:10,869: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:45:10] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-01 00:59:07,684: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-01 00:59:07,684: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-01 00:59:21,729: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:21] "GET / HTTP/1.1" 200 -]
[2023-11-01 00:59:23,969: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:23] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:23,979: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:23] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:23,992: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:23] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:23,999: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:23] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:24,034: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:24] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 00:59:24,153: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:24] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:24,237: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:24] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 00:59:42,441: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:42] "POST /predict HTTP/1.1" 200 -]
[2023-11-01 00:59:42,497: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:42] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:42,506: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:42] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 00:59:42,552: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 00:59:42] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-01 23:03:14,389: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:03:14,405: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:03:14,420: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:03:14,438: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:03:14,439: INFO: common: created directory at: artifacts]
[2023-11-01 23:03:14,440: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:03:14,441: INFO: data_ingestion: File already exists of size: ~ 23 KB]
[2023-11-01 23:03:14,460: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-01 23:03:14,460: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-01 23:03:14,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:03:14,460: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:03:14,460: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:03:14,460: INFO: common: created directory at: artifacts]
[2023-11-01 23:03:14,460: INFO: common: created directory at: artifacts/data_validation]
[2023-11-01 23:03:14,477: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/loan_data.csv']
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 26, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_02_data_validation.py", line 16, in main
    data_validation.validate_all_columns()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 35, in validate_all_columns
    raise e
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 16, in validate_all_columns
    data = pd.read_csv(self.config.unzip_data_dir)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 678, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 932, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 1216, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\common.py", line 786, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/loan_data.csv'
[2023-11-01 23:03:58,634: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:03:58,654: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:03:58,660: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:03:58,664: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:03:58,665: INFO: common: created directory at: artifacts]
[2023-11-01 23:03:58,666: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:04:01,870: INFO: data_ingestion: artifacts/data_ingestion/loan_data.zip download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 17:34:00 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"da21d70f063c2b9e71646a4f2689acf1"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=4e%2BPdWj%2BoVJlHKncFezh7UZxCl7qnf21Oac8TpnKv%2FOigjPn3sgQXzTUfBlYIMU9ANqCjoUdVlizNe92E1v0X2RzURlW5RERIdHiqG59q9%2FgBVQyfbk%2BaiCh0SHR75LeSQ5ogpTYypGsNTntRoYnT72%2B%2FdSpj5vGlIkspeSWI0mVO2Wqs6XuWcuiXP57KxXSnYNN5EjDVeeXT6uLNNwhtQsGVuIxlG7ILlbyefd91EibZn9HEWEQ4Fl0gnxSjeEVWNhzksJTGX0chZvAfFlxBA%3D%3D--zNPsSM9XakkN43Ir--LfIhCDsddb1xZIQO0dpoEA%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.21593659.1698860040; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:34:00 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:34:00 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: ED08:319F56:299F1:2E496:65428C07
connection: close

]
[2023-11-01 23:04:01,945: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:11:14,049: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:11:14,057: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:11:14,063: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:11:14,066: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:11:14,066: INFO: common: created directory at: artifacts]
[2023-11-01 23:11:14,066: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:11:14,066: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-01 23:11:14,066: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:12:53,375: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:12:53,391: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:12:53,393: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:12:53,398: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:12:53,398: INFO: common: created directory at: artifacts]
[2023-11-01 23:12:53,398: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:12:53,398: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-01 23:12:53,398: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:15:29,964: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:15:29,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:15:29,988: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:15:29,988: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:15:29,988: INFO: common: created directory at: artifacts]
[2023-11-01 23:15:29,988: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:15:34,349: INFO: data_ingestion: artifacts/data_ingestion/loan_data.csv download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 17:45:30 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"03eae56596ab1db8a0a1a8a99de668fd"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=GSxkmENmq5BUugh5j8uknKBuYkxLTWz9qANSfx1uoyxcfJGbsgG%2FPCXH5Q4YP%2BP%2BMj%2BTbtx4zpw48qCDlIxnWxN6%2BcQIjKSzBtoq5fAeVyIiI2BhEykncactz6zSVgwZqdJf3RZKpFjUR7xkeLghiwc3i4sw9u9pnT24kR%2FAIgXYW3mu7zKXUVlHUseNUMPSzewkFTcXXLHB8uPjRtTG2WjGkDZi8W4eApG5ukCmJF%2FbPBQLnz%2FhaZy2R%2BJl%2FfHwgllV8vB934DfkwIytM16Vw%3D%3D--Ae0EV9LwZ8dtZvkv--aeLE72KM4Esh0xdOR0puSw%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.959931459.1698860730; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:45:30 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:45:30 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: EF43:318874:3C21E:42F5D:65428EB9
connection: close

]
[2023-11-01 23:15:34,403: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:16:22,585: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:16:22,591: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:16:22,593: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:16:22,597: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:16:22,599: INFO: common: created directory at: artifacts]
[2023-11-01 23:16:22,599: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:16:22,600: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-01 23:16:22,601: ERROR: main: 'DataIngestion' object has no attribute 'extract_zip_file']
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
AttributeError: 'DataIngestion' object has no attribute 'extract_zip_file'
[2023-11-01 23:16:55,649: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:16:55,663: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:16:55,666: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:16:55,670: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:16:55,673: INFO: common: created directory at: artifacts]
[2023-11-01 23:16:55,676: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:16:55,676: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-01 23:16:55,676: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-01 23:16:55,676: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-01 23:16:55,676: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:16:55,688: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:16:55,692: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:16:55,694: INFO: common: created directory at: artifacts]
[2023-11-01 23:16:55,695: INFO: common: created directory at: artifacts/data_validation]
[2023-11-01 23:16:55,709: ERROR: main: Error tokenizing data. C error: Expected 1 fields in line 33, saw 6
]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 26, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_02_data_validation.py", line 16, in main
    data_validation.validate_all_columns()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 35, in validate_all_columns
    raise e
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 16, in validate_all_columns
    data = pd.read_csv(self.config.unzip_data_dir)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 678, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 581, in _read
    return parser.read(nrows)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 1253, in read
    index, columns, col_dict = self._engine.read(nrows)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\c_parser_wrapper.py", line 225, in read
    chunks = self._reader.read_low_memory(nrows)
  File "pandas\_libs\parsers.pyx", line 805, in pandas._libs.parsers.TextReader.read_low_memory
  File "pandas\_libs\parsers.pyx", line 861, in pandas._libs.parsers.TextReader._read_rows
  File "pandas\_libs\parsers.pyx", line 847, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas\_libs\parsers.pyx", line 1960, in pandas._libs.parsers.raise_parser_error
pandas.errors.ParserError: Error tokenizing data. C error: Expected 1 fields in line 33, saw 6

[2023-11-01 23:19:56,372: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:19:56,380: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:19:56,382: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:19:56,386: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:19:56,387: INFO: common: created directory at: artifacts]
[2023-11-01 23:19:56,387: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:19:57,128: INFO: data_ingestion: artifacts/data_ingestion/loan_data.csv download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 17:49:56 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"c39df8148f01168218ad564c3c4dc8d3"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=wqcp5QmdzYqp2zIhn5REyJdyqDhTmBqfCczrD2NK7RCJz8%2By4jaR5LRUS%2FOSWFKFwAVumj4%2Bct8t4Zz%2F5APs7ApSw%2FSHxtdQUGuBO2e7Fnbx1wpUHmnSK02G%2BurVQew8a5zE1A4X1wuv8ctE%2Bttj7MXK5GvMCp2ebdANyY%2B3g%2FFGo3RSgCt8AbWNZjKgThgwpWCsSkafyDVjGDI9UMM7hFTp18RkromKZUojjMnpQ8SHHLfxBqcEQoYxrWD%2FPLxHkIIaCjJ3RzZjCoqrS5G43w%3D%3D--5vM5RqvrrmguVcsd--D6he8pVPhofE80wUL7j2ZA%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.881032531.1698860996; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:49:56 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:49:56 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: F058:315759:4715C:4ED43:65428FC4
connection: close

]
[2023-11-01 23:19:57,179: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:20:32,580: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:20:32,598: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:20:32,598: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:20:32,598: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:20:32,605: INFO: common: created directory at: artifacts]
[2023-11-01 23:20:32,605: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:20:33,444: INFO: data_ingestion: artifacts/data_ingestion/loan_data.csv download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 17:50:32 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"7fbaae0cb87978ad838fe3461b7fc4df"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=H0l%2BwUD6H4B4TTceXsZ96Z3Feeb4FOMXmsxWXjOWBrfaHBZY3GXzYfF9BYLMRsoSMOxGpzvGwFyxj4cX%2FdiWWXAgAt3z1buIIFTVKPRqV0kUax2PddR3jlxYgfqGAiwylLjBOk4dljDYrIov3uX2%2BfD1kYFMOmeSeFRKwJrP%2BfftExnlMbyDVqpypMPNZ%2FiCFy%2BLl6ypo1nJnfQoMlIguwH9fZwaNT6XCRGFR5xBIGN0Bcra2bYnP0jAmH32W9ca%2FG5vJCsowZI7WwZskSAfuQ%3D%3D--F72n5y5stVMeYjZb--2A%2BEx6eGiHXwwQsqska5fA%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.182419252.1698861032; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:50:32 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:50:32 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: F06D:31B800:4C5B9:54A58:65428FE8
connection: close

]
[2023-11-01 23:20:33,498: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:21:47,547: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:21:47,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:21:47,570: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:21:47,570: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:21:47,586: INFO: common: created directory at: artifacts]
[2023-11-01 23:21:47,590: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:21:49,850: INFO: data_ingestion: artifacts/data_ingestion/data.csv download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 17:51:48 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"b72510b48fdd77fe2840c9e16ca595a4"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=o8Uxz1IKWFmShAwMXQVhM1LCDoOERq6elkjATAf0vz5fBRU5cFklw3fUv%2FvbCQ6%2BNQOTnolzAhgs4FuFQCq0tVZINMXprXFvpe%2BeOGyRraCmBe3mlGEh8qCpT9Sfz3DOJqmkdGt4FDas9Jy6u5k8TETdolmBbo%2BL5zTsqKG6GsjzUvzk5xClCASD%2BDPSA5fi84pG3Vvu43Bnu3vEqgYrxkf3Ecx1TYbsuCADHwsSKfJBRmLxbVFaMatPWlcYr1QnO3JrpgDbkcKFZYORpWh6EA%3D%3D--GlSCUCfEF5zv8mWg--QaITVh%2FdflyAZhuy3PCxpA%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.17897860.1698861107; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:51:47 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 17:51:47 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: F08E:314BE3:49AFD:51E98:65429033
connection: close

]
[2023-11-01 23:21:49,902: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:48:44,100: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-01 23:48:44,101: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-01 23:48:49,220: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:49] "GET / HTTP/1.1" 200 -]
[2023-11-01 23:48:50,651: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:50] "GET /static/js/scripts.js HTTP/1.1" 200 -]
[2023-11-01 23:48:50,655: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:50] "GET /static/css/styles.css HTTP/1.1" 200 -]
[2023-11-01 23:48:50,669: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:50] "GET /static/css2/nunito-font.css HTTP/1.1" 200 -]
[2023-11-01 23:48:50,671: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:50] "GET /static/css2/style.css HTTP/1.1" 200 -]
[2023-11-01 23:48:51,520: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:51] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 23:48:51,521: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:51] "GET /static/assets/img/form-v9.jpg HTTP/1.1" 200 -]
[2023-11-01 23:48:51,610: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:51] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 23:48:52,093: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:48:52] "GET /static/assets/favicon.ico HTTP/1.1" 200 -]
[2023-11-01 23:51:44,843: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "GET / HTTP/1.1" 200 -]
[2023-11-01 23:51:44,883: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:51:44,887: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:51:44,887: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:51:44,887: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 23:51:44,960: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 23:51:44,962: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 23:51:44,978: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 23:51:44,999: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:51:44] "[36mGET /static/assets/favicon.ico HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:02,132: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-01 23:52:02,132: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-01 23:52:08,166: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:08] "GET / HTTP/1.1" 200 -]
[2023-11-01 23:52:09,584: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:09,584: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:09,584: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:09,591: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:09,636: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-01 23:52:09,640: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-01 23:52:09,739: INFO: _internal: 192.168.0.100 - - [01/Nov/2023 23:52:09] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-01 23:59:31,247: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:59:31,259: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:59:31,259: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:59:31,275: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:59:31,275: INFO: common: created directory at: artifacts]
[2023-11-01 23:59:31,291: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:59:32,293: INFO: data_ingestion: artifacts/data_ingestion/data.csv download! with following info: 
Server: GitHub.com
Date: Wed, 01 Nov 2023 18:29:31 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"6c0ba78093c280cc11668807f9cd4e06"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.githubcopilot.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com cdn.optimizely.com logx.optimizely.com/v1/events objects-origin.githubusercontent.com *.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ wss://*.actions.githubusercontent.com github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com support.github.com; img-src 'self' data: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/
Set-Cookie: _gh_sess=8FtRrHEr7I0fSNkbHi4DnUoE2%2BVHS3JqU8gkeG4ddKnwKNavktdkzhVq19%2F8i7o9TNzsJVAi%2FRUcWvOezODzRww1N%2Fjesw4ZMLRtaomRgOhjGhFhKSlSiS%2FACwNsR3UsmR7PnJ3DJXitVWRHzNtFzc0mBBAv1p9TZmoQQ2ODAYidOfvb2xX29gOm564MEXKpHF4Z6PCqo74CGJiNsp%2BfB0R42ErwmnYNAWJ5jgqFErDo%2Fh3NGTfZ20FeUbPTZpNjB68VRDwNoZ%2BmpPmIZFB4BQ%3D%3D--mR%2FfPvHOPv%2FUiT%2Bx--ZouITyuE%2B%2BUm98mdbL506g%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.460573387.1698863371; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 18:29:31 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Fri, 01 Nov 2024 18:29:31 GMT; HttpOnly; Secure; SameSite=Lax
Accept-Ranges: bytes
Transfer-Encoding: chunked
X-GitHub-Request-Id: F628:315759:88031:97583:6542990B
connection: close

]
[2023-11-01 23:59:32,368: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-01 23:59:45,228: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-01 23:59:45,237: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-01 23:59:45,237: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-01 23:59:45,244: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-01 23:59:45,248: INFO: common: created directory at: artifacts]
[2023-11-01 23:59:45,248: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-01 23:59:45,252: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-01 23:59:45,252: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-02 00:03:18,254: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:03:18,277: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:03:18,285: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:03:18,292: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:03:18,292: INFO: common: created directory at: artifacts]
[2023-11-02 00:03:18,292: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:03:18,292: INFO: data_ingestion: File already exists of size: ~ 121 KB]
[2023-11-02 00:03:18,301: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-02 00:03:49,817: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:03:49,835: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:03:49,839: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:03:49,843: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:03:49,843: INFO: common: created directory at: artifacts]
[2023-11-02 00:03:49,843: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:03:53,082: INFO: data_ingestion: artifacts/data_ingestion/data.csv download! with following info: 
Content-Type: text/html; charset=utf-8
X-Frame-Options: DENY
Set-Cookie: __Host-GAPS=1:vanpNi25HQ5SS3kLr9GaE58v_-ZBuA:T-TDftsKJ71NMr3P; Expires=Fri, 31-Oct-2025 18:33:52 GMT; Path=/; Secure; HttpOnly; Priority=HIGH
x-auto-login: realm=com.google&args=service%3Dwise%26continue%3Dhttps://drive.google.com/open?id%253D1ZlS_09ybbDHRLnhwjYZsrgkDvr6OE030%2526usp%253Ddrive_fs
Link: <https://www.google.com/intl/en-US/drive/>; rel="canonical"
x-ua-compatible: IE=edge
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Mon, 01 Jan 1990 00:00:00 GMT
Date: Wed, 01 Nov 2023 18:33:52 GMT
Strict-Transport-Security: max-age=31536000; includeSubDomains
Content-Security-Policy: require-trusted-types-for 'script';report-uri /v3/signin/_/AccountsSignInUi/cspreport
Content-Security-Policy: script-src 'report-sample' 'nonce-JuoVQlxvJilSgab7YE2dBQ' 'unsafe-inline';object-src 'none';base-uri 'self';report-uri /v3/signin/_/AccountsSignInUi/cspreport;worker-src 'self'
Report-To: {"group":"AccountsSignInUi","max_age":2592000,"endpoints":[{"url":"https://csp.withgoogle.com/csp/report-to/AccountsSignInUi"}]}
Cross-Origin-Opener-Policy-Report-Only: same-origin; report-to="AccountsSignInUi"
Cross-Origin-Resource-Policy: same-site
Permissions-Policy: ch-ua-arch=*, ch-ua-bitness=*, ch-ua-full-version=*, ch-ua-full-version-list=*, ch-ua-model=*, ch-ua-wow64=*, ch-ua-form-factor=*, ch-ua-platform=*, ch-ua-platform-version=*
Accept-CH: Sec-CH-UA-Arch, Sec-CH-UA-Bitness, Sec-CH-UA-Full-Version, Sec-CH-UA-Full-Version-List, Sec-CH-UA-Model, Sec-CH-UA-WoW64, Sec-CH-UA-Form-Factor, Sec-CH-UA-Platform, Sec-CH-UA-Platform-Version
Server: ESF
X-XSS-Protection: 0
X-Content-Type-Options: nosniff
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
Accept-Ranges: none
Vary: Sec-Fetch-Dest, Sec-Fetch-Mode, Sec-Fetch-Site,Accept-Encoding
Connection: close
Transfer-Encoding: chunked

]
[2023-11-02 00:03:53,399: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-02 00:05:23,453: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:05:23,477: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:05:23,477: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:05:23,477: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:05:23,477: INFO: common: created directory at: artifacts]
[2023-11-02 00:05:23,484: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:05:25,122: INFO: data_ingestion: artifacts/data_ingestion/data.csv download! with following info: 
Content-Type: text/html; charset=utf-8
X-Robots-Tag: noindex, nofollow, nosnippet
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Mon, 01 Jan 1990 00:00:00 GMT
Date: Wed, 01 Nov 2023 18:35:24 GMT
P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
Content-Security-Policy: require-trusted-types-for 'script';report-uri https://csp.withgoogle.com/csp/docs-tt
Referrer-Policy: origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Server: GSE
Set-Cookie: NID=511=G_CPP_1ndZ7K9jvzdhBkREpvPOdV7HO4FL2ecD0AGZLZBNbJPyiZhPsTOG3ulPE6kya6m4b6K8Nm28CwOIfkJdOfTIN438cEX_KorOw0dJ1Z9BI6lNa93MseLCiwSty3WpwjafPB2uxhNL4XucmDhTt9EIOQvgUiWyy4OTvZrrQ; expires=Thu, 02-May-2024 18:35:23 GMT; path=/; domain=.google.com; HttpOnly
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
Accept-Ranges: none
Vary: Sec-Fetch-Dest, Sec-Fetch-Mode, Sec-Fetch-Site,Accept-Encoding
Connection: close
Transfer-Encoding: chunked

]
[2023-11-02 00:05:25,345: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 13, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_01_data_ingestion.py", line 18, in main
    data_ingestion.extract_zip_file()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1257, in __init__
    self._RealGetContents()
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\zipfile.py", line 1324, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-11-02 00:10:35,793: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:10:35,824: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:10:35,824: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:10:35,824: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:10:35,824: INFO: common: created directory at: artifacts]
[2023-11-02 00:10:35,824: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:10:37,419: INFO: data_ingestion: artifacts/data_ingestion/loan_data.csv download! with following info: 
Content-Type: text/html; charset=utf-8
X-Robots-Tag: noindex, nofollow, nosnippet
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Mon, 01 Jan 1990 00:00:00 GMT
Date: Wed, 01 Nov 2023 18:40:36 GMT
P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
Content-Security-Policy: require-trusted-types-for 'script';report-uri https://csp.withgoogle.com/csp/docs-tt
Referrer-Policy: origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Server: GSE
Set-Cookie: NID=511=gqo0BLDTWN3nutmoUz8JoaakOn37YcxnChQBUqFfioykw92aIjEHCY_mUxd686_NTgjX50eBvVl-h9HdS3ZEmketh1nmc_Uocnf4ttNGIdEX4IBwj-ePRK5LJqHhSG3C3Wu5Rre_T3fHYjgMMmnyahqaIKrTIg0JmsQwEx3CTco; expires=Thu, 02-May-2024 18:40:36 GMT; path=/; domain=.google.com; HttpOnly
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
Accept-Ranges: none
Vary: Sec-Fetch-Dest, Sec-Fetch-Mode, Sec-Fetch-Site,Accept-Encoding
Connection: close
Transfer-Encoding: chunked

]
[2023-11-02 00:10:37,419: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:10:37,434: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:10:37,435: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:10:37,442: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:10:37,442: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:10:37,442: INFO: common: created directory at: artifacts]
[2023-11-02 00:10:37,452: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:10:38,353: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:10:38,353: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:10:38,370: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:10:38,377: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:10:38,377: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:10:38,377: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:10:38,385: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:10:38,385: INFO: common: created directory at: artifacts]
[2023-11-02 00:10:38,385: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:10:38,385: ERROR: main: "'ConfigBox' object has no attribute 'name'"]
Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\config_box.py", line 28, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 49, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_04_model_trainer.py", line 15, in main
    model_trainer_config = config.get_model_trainer_config()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\config\configuration.py", line 84, in get_model_trainer_config
    target_column = schema.name
  File "box\config_box.py", line 30, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"
[2023-11-02 00:17:17,656: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:17:17,671: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:17:17,678: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:17:17,678: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:17:17,678: INFO: common: created directory at: artifacts]
[2023-11-02 00:17:17,685: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:17:17,685: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:17:17,685: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:17:17,693: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:17:17,698: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:17:17,700: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:17:17,704: INFO: common: created directory at: artifacts]
[2023-11-02 00:17:17,704: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:17:17,805: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:17:17,805: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:17:17,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:17:17,820: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:17:17,820: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:17:17,820: INFO: common: created directory at: artifacts]
[2023-11-02 00:17:17,820: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-02 00:17:18,299: INFO: data_transformation: Splited data into training and test sets]
[2023-11-02 00:17:18,299: INFO: data_transformation: (22500, 8)]
[2023-11-02 00:17:18,299: INFO: data_transformation: (7500, 8)]
[2023-11-02 00:17:18,299: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:17:18,299: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:17:18,313: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:17:18,313: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:17:18,320: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:17:18,320: INFO: common: created directory at: artifacts]
[2023-11-02 00:17:18,324: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:17:18,326: ERROR: main: "'ConfigBox' object has no attribute 'name'"]
Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\config_box.py", line 28, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 49, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_04_model_trainer.py", line 15, in main
    model_trainer_config = config.get_model_trainer_config()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\config\configuration.py", line 84, in get_model_trainer_config
    target_column = schema.name
  File "box\config_box.py", line 30, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"
[2023-11-02 00:19:35,127: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:19:35,134: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:19:35,141: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:19:35,158: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:19:35,165: INFO: common: created directory at: artifacts]
[2023-11-02 00:19:35,165: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:19:35,165: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:19:35,165: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:19:35,177: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:19:35,177: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:19:35,189: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:19:35,189: INFO: common: created directory at: artifacts]
[2023-11-02 00:19:35,189: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:19:35,268: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:19:35,268: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:19:35,268: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:19:35,268: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:19:35,285: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:19:35,285: INFO: common: created directory at: artifacts]
[2023-11-02 00:19:35,285: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-02 00:19:35,792: INFO: data_transformation: Splited data into training and test sets]
[2023-11-02 00:19:35,792: INFO: data_transformation: (22500, 8)]
[2023-11-02 00:19:35,806: INFO: data_transformation: (7500, 8)]
[2023-11-02 00:19:35,808: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:19:35,810: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:19:35,810: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:19:35,810: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:19:35,810: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:19:35,821: INFO: common: created directory at: artifacts]
[2023-11-02 00:19:35,824: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:19:35,824: ERROR: main: "'ConfigBox' object has no attribute 'name'"]
Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\config_box.py", line 28, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 49, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_04_model_trainer.py", line 15, in main
    model_trainer_config = config.get_model_trainer_config()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\config\configuration.py", line 84, in get_model_trainer_config
    target_column = schema.name
  File "box\config_box.py", line 30, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"
[2023-11-02 00:25:48,442: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:25:48,450: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:25:48,450: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:25:48,458: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:25:48,458: INFO: common: created directory at: artifacts]
[2023-11-02 00:25:48,458: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:25:48,458: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:25:48,458: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:25:48,466: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:25:48,473: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:25:48,473: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:25:48,473: INFO: common: created directory at: artifacts]
[2023-11-02 00:25:48,473: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:25:48,571: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:25:48,579: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:25:48,587: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:25:48,589: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:25:48,589: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:25:48,589: INFO: common: created directory at: artifacts]
[2023-11-02 00:25:48,589: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-02 00:25:49,097: INFO: data_transformation: Splited data into training and test sets]
[2023-11-02 00:25:49,105: INFO: data_transformation: (22500, 8)]
[2023-11-02 00:25:49,105: INFO: data_transformation: (7500, 8)]
[2023-11-02 00:25:49,105: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:25:49,105: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:25:49,113: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:25:49,121: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:25:49,121: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:25:49,129: INFO: common: created directory at: artifacts]
[2023-11-02 00:25:49,129: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:25:49,132: ERROR: main: "'ConfigBox' object has no attribute 'name'"]
Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\config_box.py", line 28, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 503, in box.box.Box.__getitem__
KeyError: 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\box.py", line 536, in box.box.Box.__getattr__
  File "box\box.py", line 524, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'name'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\box.py", line 538, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 49, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_04_model_trainer.py", line 15, in main
    model_trainer_config = config.get_model_trainer_config()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\config\configuration.py", line 84, in get_model_trainer_config
    target_column = schema.name
  File "box\config_box.py", line 30, in box.config_box.ConfigBox.__getattr__
  File "box\box.py", line 552, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'name'"
[2023-11-02 00:38:23,339: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:38:23,346: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:38:23,350: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:38:23,364: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:38:23,370: INFO: common: created directory at: artifacts]
[2023-11-02 00:38:23,370: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:38:23,370: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:38:23,370: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:38:23,378: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:38:23,386: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:38:23,394: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:38:23,394: INFO: common: created directory at: artifacts]
[2023-11-02 00:38:23,399: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:38:23,489: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:38:23,497: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:38:23,497: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:38:23,505: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:38:23,505: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:38:23,505: INFO: common: created directory at: artifacts]
[2023-11-02 00:38:23,505: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-02 00:38:23,981: INFO: data_transformation: Splited data into training and test sets]
[2023-11-02 00:38:23,981: INFO: data_transformation: (22500, 8)]
[2023-11-02 00:38:23,981: INFO: data_transformation: (7500, 8)]
[2023-11-02 00:38:23,990: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:38:23,990: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:38:23,990: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:38:23,998: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:38:24,000: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:38:24,000: INFO: common: created directory at: artifacts]
[2023-11-02 00:38:24,000: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:38:24,275: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2023-11-02 00:38:24,275: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2023-11-02 00:38:24,283: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:38:24,293: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:38:24,293: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:38:24,293: INFO: common: created directory at: artifacts]
[2023-11-02 00:38:24,301: INFO: common: created directory at: artifacts/model_evaluation]
[2023-11-02 00:38:24,766: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2023-11-02 00:38:39,065: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2023-11-02 00:39:04,849: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 00:39:04,849: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 00:39:09,501: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:09] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:39:11,220: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:39:11,230: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:39:11,240: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:39:11,240: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:39:11,281: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:39:11,289: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:39:11,402: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 00:39:11,483: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:39:11] "[36mGET /static/assets/favicon.ico HTTP/1.1[0m" 304 -]
[2023-11-02 00:41:52,606: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:41:52] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 00:41:52,684: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:41:52] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2023-11-02 00:45:13,118: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 00:45:13,118: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 00:45:19,453: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:19] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:45:21,097: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:45:21,101: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:45:21,105: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:45:21,113: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:45:21,155: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:45:21,252: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:45:21,310: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:21] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 00:45:35,768: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:45:35] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 00:47:12,984: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 00:47:12,984: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 00:47:30,941: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:30] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:47:32,977: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:32] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:47:32,977: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:32] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:47:32,984: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:32] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:47:32,993: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:32] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:47:33,904: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:33] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:47:33,913: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:33] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:47:34,113: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:34] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 00:47:45,791: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:45] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 00:47:45,911: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:47:45] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2023-11-02 00:48:38,702: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:48:38,721: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:48:38,721: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:48:38,721: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:48:38,721: INFO: common: created directory at: artifacts]
[2023-11-02 00:48:38,721: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:48:38,721: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:48:38,734: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:48:38,734: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:48:38,734: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:48:38,772: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:48:38,794: INFO: common: created directory at: artifacts]
[2023-11-02 00:48:38,802: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:48:38,828: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/loan_data.csv']
Traceback (most recent call last):
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\main.py", line 26, in <module>
    data_ingestion.main()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\pipeline\stage_02_data_validation.py", line 16, in main
    data_validation.validate_all_columns()
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 35, in validate_all_columns
    raise e
  File "c:\Users\kshit\OneDrive\Documents\loan sanction\mlProject\components\data_validation.py", line 16, in validate_all_columns
    data = pd.read_csv(self.config.unzip_data_dir)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\util\_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 678, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 932, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\parsers\readers.py", line 1216, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "C:\Users\kshit\AppData\Local\Programs\Python\Python39\lib\site-packages\pandas\io\common.py", line 786, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/loan_data.csv'
[2023-11-02 00:49:59,974: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-11-02 00:49:59,988: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:49:59,992: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:49:59,996: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:49:59,996: INFO: common: created directory at: artifacts]
[2023-11-02 00:50:00,014: INFO: common: created directory at: artifacts/data_ingestion]
[2023-11-02 00:50:00,021: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-11-02 00:50:00,043: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-11-02 00:50:00,080: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:50:00,080: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:50:00,090: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:50:00,090: INFO: common: created directory at: artifacts]
[2023-11-02 00:50:00,096: INFO: common: created directory at: artifacts/data_validation]
[2023-11-02 00:50:00,214: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-11-02 00:50:00,222: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-11-02 00:50:00,230: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:50:00,230: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:50:00,238: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:50:00,238: INFO: common: created directory at: artifacts]
[2023-11-02 00:50:00,238: INFO: common: created directory at: artifacts/data_transformation]
[2023-11-02 00:50:00,695: INFO: data_transformation: Splited data into training and test sets]
[2023-11-02 00:50:00,695: INFO: data_transformation: (22500, 7)]
[2023-11-02 00:50:00,695: INFO: data_transformation: (7500, 7)]
[2023-11-02 00:50:00,695: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-11-02 00:50:00,695: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-11-02 00:50:00,695: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:50:00,710: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:50:00,713: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:50:00,713: INFO: common: created directory at: artifacts]
[2023-11-02 00:50:00,718: INFO: common: created directory at: artifacts/model_trainer]
[2023-11-02 00:50:00,940: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2023-11-02 00:50:00,940: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2023-11-02 00:50:00,947: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-11-02 00:50:00,947: INFO: common: yaml file: params.yaml loaded successfully]
[2023-11-02 00:50:00,955: INFO: common: yaml file: schema.yaml loaded successfully]
[2023-11-02 00:50:00,966: INFO: common: created directory at: artifacts]
[2023-11-02 00:50:00,966: INFO: common: created directory at: artifacts/model_evaluation]
[2023-11-02 00:50:01,407: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2023-11-02 00:50:12,386: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2023-11-02 00:50:36,103: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 00:50:36,107: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 00:50:38,968: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:38] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:50:40,753: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:40,753: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:40,753: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:40,770: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:40,830: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:50:40,953: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:40] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:41,033: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:41] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 00:50:55,076: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:55] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 00:50:55,166: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:55] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:55,166: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:55] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:50:55,228: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:50:55] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-02 00:56:20,215: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 00:56:20,215: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 00:56:27,973: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:27] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:56:29,799: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:29] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:29,799: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:29] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:29,815: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:29] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:29,815: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:29] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:29,871: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:29] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:56:30,000: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:30] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:30,065: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:30] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 00:56:43,454: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:43] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 00:56:43,540: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:43] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:43,558: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:43] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:56:43,628: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:56:43] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-02 00:58:51,256: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "GET / HTTP/1.1" 200 -]
[2023-11-02 00:58:51,336: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:58:51,352: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:58:51,352: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 00:58:51,368: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 00:58:51,424: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 00:58:51,476: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 00:58:51,495: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 00:58:51] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 01:00:13,191: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:00:13] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 01:00:13,280: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:00:13] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 01:00:13,288: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:00:13] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 01:00:13,336: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:00:13] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
[2023-11-02 01:02:24,680: WARNING: _internal:  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.]
[2023-11-02 01:02:24,680: INFO: _internal:  * Running on http://192.168.0.100:8080/ (Press CTRL+C to quit)]
[2023-11-02 01:02:30,642: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:30] "GET / HTTP/1.1" 200 -]
[2023-11-02 01:02:32,488: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:32] "[36mGET /static/css2/nunito-font.css HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:32,488: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:32] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:32,496: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:32] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:32,496: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:32] "[36mGET /static/css2/style.css HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:33,578: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:33] "[33mGET /static/assets/img/wine.jpg HTTP/1.1[0m" 404 -]
[2023-11-02 01:02:33,583: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:33] "[36mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:33,742: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:33] "[33mGET /static/fonts/Nunito/Nunito-Bold.ttf HTTP/1.1[0m" 404 -]
[2023-11-02 01:02:45,820: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:45] "POST /predict HTTP/1.1" 200 -]
[2023-11-02 01:02:45,944: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:45] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:45,951: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:45] "[36mGET /static/js/scripts.js HTTP/1.1[0m" 304 -]
[2023-11-02 01:02:46,121: INFO: _internal: 192.168.0.100 - - [02/Nov/2023 01:02:46] "[33mGET /static/assets/img/wine.gif HTTP/1.1[0m" 404 -]
